surefire-booter/src/main/java/org/apache/maven/surefire/booter/ForkedBooter.java

```diff
@@ -236,13 +236,17 @@ private static void exit( int returnCode, Shutdown shutdownType, CommandReader r
         switch ( shutdownType )
         {
             case KILL:
+                System.out.close();
```

Comment:
@ChristianSchulte 
This is executed right before JVM exit.
But our problem is in the beginning of life time of JVM.
This means the `InputStream.read()` and (`Process.waitFor()` or `Process.exec()`) are mutual exclusive on FreeBSD according to logs.
It seems std/err will be tested instead of std/out.
Additionally ACK before shutdown needs to be added as StephenC mentioned.